test/fileHandler/index_test.js

```diff
@@ -610,6 +610,113 @@ describe('#fileHandler', function () {
 				}
 			}
 		});
+
+		it(`should upload file with access control list`, async () => {
+			const testFilePath = '/tmp/falafel/tests/example.txt';
+			fs.ensureFileSync(testFilePath);
+			fs.writeFileSync(testFilePath, 'Example content');
+			const contentLength = Buffer.byteLength('Example content', 'utf8');
+
+			const randomGuid = guid();
+			getProxiedFileHandler({
+				'mout/random/guid': () => {
+					return randomGuid;
+				},
+				'aws-sdk': {
+					'S3': class S3 {
+						async upload (uploadParams, uploadOptions, callback) {
+							assert.strictEqual(uploadParams.ACL, 'bucket-owner-full-control');
+							callback(null, {
+								Bucket: uploadParams.Bucket,
+								Key: uploadParams.Key,
+							});
+						}
+						getSignedUrl (operation, signedParams, callback) {
+							callback(null, 'https://test.aws.com/buckethash');
+						}
+					}
+				}
+			});
+
+			const uploadResult = await falafel.files.upload({
+				contentType: 'text/plain',
+				name: 'example.txt',
+				length: contentLength,
+				file: testFilePath
+			});
+		});
+
+		it(`should upload file with acl flag set as false`, async () => {
+			const testFilePath = '/tmp/falafel/tests/example.txt';
+			fs.ensureFileSync(testFilePath);
+			fs.writeFileSync(testFilePath, 'Example content');
+			const contentLength = Buffer.byteLength('Example content', 'utf8');
+
+			const randomGuid = guid();
+			getProxiedFileHandler({
+				'mout/random/guid': () => {
+					return randomGuid;
+				},
+				'aws-sdk': {
+					'S3': class S3 {
+						async upload (uploadParams, uploadOptions, callback) {
+							assert.strictEqual(uploadParams.ACL, 'bucket-owner-full-control');
+							callback(null, {
+								Bucket: uploadParams.Bucket,
+								Key: uploadParams.Key,
+							});
+						}
+						getSignedUrl (operation, signedParams, callback) {
+							callback(null, 'https://test.aws.com/buckethash');
+						}
+					}
+				}
+			});
+
+			const uploadResult = await falafel.files.upload({
+				contentType: 'text/plain',
+				name: 'example.txt',
+				length: contentLength,
+				file: testFilePath,
+				disableObjectAcl: false
```

Comment:
do we wanna add this property to readme somewhere? feel like it should be documented

Reply:
Good point. Have added a description.